mentor-mind-app/src/pages/SettingsPage.tsx

```diff
@@ -0,0 +1,20 @@
+import {FormLabel, Stack, TextField} from "@mui/material";
+import PinnedContainer from "../components/containers/PinnedContainer";
+import RegisterComponent from "../components/login-register/RegisterComponent";
+import LogInComponent from "../components/login-register/LogInComponent";
+import SettingsMainComponent from "../components/settings/SettingsMain";
+import SettingsPassword from "../components/settings/SettingsPassword";
+
+const SettingsPage = () => {
+    return (
+        <Stack direction={"column"} justifyContent={"space-evenly"} minHeight={"calc( 100% - 100px );"} height={"calc( 100% - 100px );"} alignItems={"center"} margin={"auto"}>
```

Comment:
To ensure that the SettingsPassword  form is a bit above the bottom of the page, feel free to add the following attribute to this stack:
sx = {{ pb: //size }}
This will add bottom padding to the element